src/components/document-list/_macro.spec.js

```diff
@@ -366,7 +314,7 @@ describe('FOR: Macro: Document list', () => {
 
     describe('GIVEN: Params: showprefix', () => {
         describe('WHEN: showprefix is set in the date metadata configuration for a document', () => {
-            test('THEN: has the correct class', () => {
+            test('THEN: applies the prefix class to the document item', () => {
```

Comment:
This isn't testing that a prefix class is being added. Its testing that the font is bold. I think we need to be testing that it has that class and doesn't have the `ons-u-vh` class